package.json

```diff
@@ -59,10 +59,12 @@
     "eslint-config-prettier": "^4.1.0",
     "eslint-plugin-react": "^7.12.4",
     "jest": "^24.5.0",
+    "jest-dom": "^3.1.3",
     "mongodb-memory-server": "^4.2.1",
     "nodemon": "^1.18.10",
     "pino-pretty": "^2.6.0",
     "prettier": "^1.16.4",
-    "react-test-renderer": "^16.8.5"
+    "react-test-renderer": "^16.8.5",
+    "react-testing-library": "^6.0.4"
```

Comment:
I don't have a deep understanding of testing libraries, but why do we have to use both `enzyme` and `react-testing-library`?

Reply:
React Hooks are not supported in Enzyme yet, but they are in react-testing-library. However, I think Enzyme is easier to understand for testing component renderings. We could use only react-testing-library, but I don't see a problem with using them both. The important thing is that the tests run automatically and are written to isolate errors/failures well enough that the bad code can be quickly identified.

Reply:
I see. Thank you for the info.